recipes/jxrlib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jxrlib" %}
+{% set version = "1.1" %}
+{% set commit = "98e615dc5b08d684a270800aa631e3511b8c3efa" %}
+{% set sha256 = "d10532947f9f4beb0b67fa2ad064c6ec5edf05281f4a0be0904d9527e37acd4a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.zip
+  url: https://github.com/glencoesoftware/{{ name }}/archive/{{ commit }}.zip
```

Comment:
This appears to be someone's mirror unless I'm mistaken.

FWICT [this is the official copy]( https://archive.codeplex.com/?p=jxrlib ). Though downloading this gives a zip that is mostly unusable. The files all have `\` in their names as if they were meant to specify paths and a bunch of miscellaneous content related to the issue tracker, discussions, and various other things comes with. 

Perhaps we can use something like the [Debian rehosted source code]( https://packages.debian.org/source/sid/jxrlib ), which is closer to the official source and better organized. WDYT?

Reply:
IIRC, glencoesoftware was contracted by ZEISS and OME/BioFormats to make the original jxrlib (by Microsoft) usable for BioFormat's CZI reader. At that point I switched from the original jxrlib to glencoesoftware's + my `CreateDecoderFromBytes` patch. Works well for me.

Reply:
Ok, sounds like you know way more about the details here. Happy to defer to your judgement.

Reply:
SGTM. Thanks all.